index.html

```diff
@@ -2888,11 +2900,59 @@ <h3><dfn>Writing or pushing content</dfn></h3>
       </p>
       </section>
 
+      <section><h3>Mapping external data to NDEF</h3>
```

Comment:
I guess user data would be more appropriate than "external" data. The work external is a bit overloaded in this context.

Reply:
I thought about user or custom, but I actually find this more clear